Automattic/jetpack
# Packages: Constants_Manager -> Constants

During Today's call, democracy talked an decided in favour of having more concise and ergonomic names.

#### Changes proposed in this Pull Request:
* Refactors `Constatns_Manager` into `Constants`
* Constants 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Part of the Jetpack DNA project - p1HpG7-70O-p2

#### Testing instructions:
* Checkout this branch.
* Run `rm -rf ./vendor` inside Jetpack.
* Run `composer install` inside Jetpack.
* Verify that pass.
* Verify CI passes on this PR and 100% is green.
* Smoke test Jetpack to verify it still works well in both the admin and the frontend.
